sample/util/SetUpCredentials.php

```diff
@@ -0,0 +1,57 @@
+<?php
+require_once('vendor/autoload.php');
+
+use Bynder\Api\BynderClient;
+use Bynder\Api\Impl\OAuth2;
+
+class SetUpCredentials
+{
```

Comment:
This could also have been a simple function, but then again the whole SDK is basically "must OO everything" 😀

Reply:
Yeah that's true. I tried to stick to that I guess.